darshan-util/pydarshan/darshan/report.py

```diff
@@ -698,7 +722,7 @@ def mod_read_all_apmpi_records(self, mod="APMPI", dtype=None, warnings=True):
             self.update_name_records(mod=mod)
 
 
-    def mod_read_all_apxc_records(self, mod="APXC", dtype=None, warnings=True):
+    def mod_read_all_apxc_records(self, mod="APXC", dtype=None, _warnings=True):
```

Comment:
I assume @shanedsnyder is happy with the backwards incompatible function signature change in support of the deprecation warnings for `dtype`.

Reply:
To be clear, you just mean the change of `warnings` to `_warnings`? Or something else?

If the former, I don't think it's a big deal, but not sure it's really needed (seems like you have some questions on this change above, too).